config/routes.rb

```diff
@@ -425,10 +427,14 @@
         show_list
         tagging_edit
         tag_edit_form_field_changed
+        backup_create
+        backup_restore
```

Comment:
I wonder why do we need both `GET` and `POST` routes for these? Why can't we just use on of them?